src/main/java/de/tum/in/www1/artemis/service/connectors/bamboo/BambooTriggerService.java

```diff
@@ -47,4 +48,10 @@ public void triggerBuild(ProgrammingExerciseParticipation participation) throws
             throw new BambooException("Communication failed when trying to trigger the Bamboo build plan " + buildPlan + " with the error: " + e.getMessage());
         }
     }
+
+    @Override
+    public void triggerBuild(ProgrammingExerciseParticipation participation, String commitHash) throws ContinuousIntegrationException {
+        log.warn("Triggering builds with a commit hash is not supported for Bamboo. Triggering build without commit hash.");
```

Comment:
This message is a bit misleading. Maybe in every implementatiation we should have standardised:
```java
log.warn("Triggering builds with a commit hash: hash=${HASH, ci=BAMBOO|LOCAL\etc..}.");
```

And then for those not supporting it a second log in a consecutive line:
```java
log.warn("Triggering builds with a commit hash is not supported for Bamboo. Triggering build without commit hash.");
```

so that is is clear where this warning comes from

Reply:
It should be solved now. The method was removed as it was unnecessary.